graphlient.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.homepage = 'http://github.com/ashkan18/graphlient'
   s.licenses = ['MIT']
   s.summary = 'A friendlier Ruby client for consuming GraphQL-based APIs.'
-  s.add_dependency 'faraday'
+  s.add_dependency 'faraday', '~> 1.0'
```

Comment:
This should be `>= 1.0`, there's no reason a priori why Faraday 2.0 wouldn't work until it doesn't.